pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -76,6 +79,14 @@ def get_unit_data(self, unit):
         if unit['submitted_by__email']:
             email_md5 = md5(unit['submitted_by__email']).hexdigest()
 
+        iso_submitted = None
```

Comment:
I recall this was `iso_submitted = unit.get('submitted_on', None)` before, which is the preferred idiomatic Python way to set default values. Also I realize the `_on` from the original variable is missing here and in the `display_` variant.